communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,223 @@
+        <div>
+          <img src="{{ figure_microbiome_url }}" alt="Overview of microbial data analysis" style="max-width: 100%;">
+          <p style="font-size: 0.9em;">
+
+            Microbiome samples encompass metabarcoding, metagenomics, metatranscriptomics, and metaproteomics with corresponding tool suites and potential analyses available on microGalaxy.
+
+            The workflow progresses from raw data to intermediate and final outputs. Outputs are color-coded by type, with numbers indicating tool suites used for each step:
+
+            <strong> Input data (blue)</strong> represent raw datasets, including sequencing reads and mass spectra.  
+            <strong> Intermediate outputs (gray)</strong> include cleaned reads, assemblies, and OTUs/AVSs.  
+            <strong> Taxonomic outputs (green)</strong> represent taxonomic profiling results.  
+            <strong> Functional outputs (yellow)</strong> highlight functional profiling and differential feature analysis.  
+            <strong> Other outputs (orange)</strong> indicate annotated genomes, reconstructed genomes, and antibiotic resistance genes.
+
+            <strong> Numbers correspond to tool suites:</strong>  
+            (1) FastQC, Cutadapt, fastp, MultiQC, Porechop, Fitlong, Nanoplot, SortMeRNA, Bowtie, minimap  
+            (2) MetaPhlAn, Kraken(2), Bracken, Diamond, Sylph, mOTUs, (Re)Centrifuge, GTDB-Tk, CLARK, Kaiju  
+            (4) Bakta, Prokka, Prodigal, DFAST, GeneMark  
+            (5) HMMER  
+            (6) Bakta, AMRFinderPlus, ABRicate, fARGene, hAMRonization, argNorm, StarAMR, TB-Profiler  
+            (7) ABRicate, PATRIC, VirulenceFinder  
+            (8) ISEScan, ISfinder, ISMapper, Transit, ESSENTIALS, MobileElementFinder, IslandViewer, PlasmidFinder  
+            (9) StarAMR, chewBBaca, pyMLST, MLST, SISTR  
+            (10) CRISPRCasFinder  
+            (11) antiSMASH, HMMER  
+            (12) Gene Ontology, KEGG, STRING  
+            (21) msconvert, OpenMS  
+            (22) SearchGUI, PeptideShaker, MaxQuant, FragPipe  
+            (23) MaxQuant, PepQuery, FragPipe, TMT, iTRAQ, UniPept  
+            (24) MSStats  
+            (25) QIIME2, Dada2, LOTUs, Mothur, mOTUs, FROGs  
+            (26) Picrust  
+            (27) MaAsLin2, edgeR, ANCOM, ALDEx2  
+            (28) Krakentools  
+            (29) KRONA, GraPhlAn, Pavian, Phinch  
+            (30) HUMAnN, minimap, BWA-MEM Diamond  
+            (31) Groot, deepARG  
+            (32) MEGAHIT, MetaSPades, metaFlye, Unicycler, Racon, Pilon, Polypolish, (meta)QUAST  
+            (33) MetaBAT, CONCOT, MAxBin, SemiBin, DASTool, dRep, CoverM, CheckM  
+            (34) GTDB-Tk  
+            (35) Metanovo  
+            (36) Unipept  
+            (37) MetaQuantome
+          </p>
+        </div>
+
+- id: tools
+  title: Tools
+  heading_md: >
+    Flagship tools for microbiome analysis
+  content:
+    - title_md: <code>racon</code> - A standalone consensus building tool
```

Comment:
It might be nice to use subsections (metabarcoding, metagenomics, metatranscriptomics, metaproteomics) here and highlight 2-3 tools per subsections.
Example of subsection: https://labs.usegalaxy.org.au/ (bottom)

Reply:
e.g. https://github.com/usegalaxy-au/galaxy-labs-engine/blob/dev/app/labs/static/labs/content/docs/section_3.yml
https://labs.usegalaxy.org.au/

Reply:
add more database, Bakta, prokka, nt